app/classifier/classifier.jsx

```diff
@@ -272,7 +272,8 @@ class Classifier extends React.Component {
         finished_workflow,
         retired,
         selection_state,
-        user_has_finished_workflow
+        user_has_finished_workflow,
+        selected_at
```

Comment:
I assume this will be ok if the `selected_at` attribute doesn't exist on the subject.

Reply:
Should be. You might see classifications with `selected_at` set to 'undefined'.